app/views/users/show.html.erb

```diff
@@ -72,14 +72,15 @@
     </section>
   <% end %>
 
-  <% if current_user && current_user.practices.any? %>
+  <% created_practices = Practice.where(user: current_user) %>
```

Comment:
Question for learning purposes 😅: Is there a preference for one variables should be set in the controller vs the view? 🤔

Reply:
good question.
I don't have a system for this...
I think typically you're only supposed to have one "instance" variable
(that's the variables prefaced with `@`)
However, setting things in the view fixes that problem.
I don't mind either way as long as code is readable (or obvious) and/or explained or commented out :wink: